app/helpers/govspeak_helper.rb

```diff
@@ -253,7 +253,7 @@ def build_govspeak_document(govspeak, images = [])
 
   class AssetHostDecorator < SimpleDelegator
     def url(*args)
-      (Whitehall.asset_root || "") + super(*args)
+      (Whitehall.public_asset_host || "") + super(*args)
```

Comment:
Would be tidier to push the normalisation of `public_asset_host` into that method itself, although I see this would break  a trend elsewhere :see_no_evil: